pkg/controllers/job/plugins/ssh/ssh.go

```diff
@@ -48,6 +49,9 @@ type sshPlugin struct {
 
 	// public key string
 	sshPublicKey string
+
+	// ssh port
+	port int
```

Comment:
I donot think this works for the case required. This a global option, so if a user is using hostnetwork, any fixed port can conflict with another pod within same node

Reply:
Do you have any suggestions? 
I think they have planned the ports of the pods when using the hostNetwork.

Reply:
This is a little more complex than just set a ssh port. We need an API to set host network. 

With both port and hostnetwork set, then we still need to control pods belong to the same Job should be scheduled to different nodes(because IIUC the ssh port is per Job config).

Reply:
I think using anti-affinity can achieve this requirement, if pod uses hostNetwork, we add the same label to the pod, and then configure anti-affinity for them.